src/components/Card/Card.tsx

```diff
@@ -45,8 +48,54 @@ function Card({
     };
   }, [cardholderRef]);
 
+  const [springValues, springRef] = useSpring(
+    () => ({
+      x: 0,
+      y: 0,
+      config: {
+        mass: 2,
+        friction: 14,
+        tension: 120,
+      },
+    }),
+    []
+  );
+
+  const handleMouseLeave = () => {
+    if (cardholderRef.current) {
+      const regex =
+        /translate\((?<offsetX>[-\d.]+)px,\s*(?<offsetY>[-\d.]+)px\)/;
+      const transformValue = cardholderRef.current.style.transform;
+      const matches = regex.exec(transformValue);
+
+      if (matches?.groups) {
+        const { offsetX, offsetY } = matches.groups;
```

Comment:
maybe just add default values in case something goes wrong `const { offsetX = '0', offsetY = '0' } = matches.groups;`